libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/PasscodeActivity.java

```diff
@@ -271,74 +369,44 @@ protected TextView getTitleView() {
         return (TextView) findViewById(R.id.sf__passcode_title);
     }
 
-    protected TextView getForgotPasscodeView() {
-        return (TextView) findViewById(R.id.sf__passcode_forgot);
-    }
-
-    protected TextView getErrorView() {
-        return (TextView) findViewById(R.id.sf__passcode_error);
-    }
-
     protected TextView getInstructionsView() {
         return (TextView) findViewById(R.id.sf__passcode_instructions);
     }
 
-    protected EditText getEntryView() {
-        return (EditText) findViewById(R.id.sf__passcode_text);
+    protected PasscodeField getPasscodeField() {
```

Comment:
Remove all the new `protected` methods. We don't want apps to override this way - we want them to override through XML. These methods were created because we weren't a library project a long time ago, i.e. legacy reasons.